features/cloud/test/test_only_one_kernel.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+import helper.utils as utils
+
+def test_number_installed_kernels(client):
+    """ Only one kernel should be installed """
+    installed_kernels = utils.get_installed_kernel_versions(client)
+    nr_kernels = len(installed_kernels)
+    assert nr_kernels == 1, f"Number of installed kernels {nr_kernels} != 1"
```

Comment:
super small nitpick, but 'nr' does not seem to be a common abbreviation for 'number' in English.. I'd prefer the longer number of instead.

```suggestion
    number_of_kernels = len(installed_kernels)
    assert number_of_kernels == 1, f"Number of installed kernels {nr_kernels} != 1"
```